packages/connect-query/jest.config.ts

```diff
@@ -16,7 +16,8 @@ import type { Config } from 'jest';
 
 const config: Config = {
   preset: '../../jest-preset.js',
-  testEnvironment: 'jsdom',
+  // testEnvironment: 'jsdom',
```

Comment:
Maybe delete comment... Or keep it, up to you.

Reply:
good call, I missed this.